### PR TITLE
LWIMIQ for 2-21

### PR DESCRIPTION
**Pull request by AllenBW**
Oh hey guyz, don't be mental, its my first :hocho:

 @syncrou @JPrause @imtayadeway ya'll seem to be da :eyes: that make all things :godmode:

### CODE REVIEW

File: site/_posts/2017-02-20-februarys-final-weeks-work.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Woah first last week of February for 2017, this is :taco::dancer: **huge**. *Almost* as :taco::dancer: huge as the LQIMIQ low down prepared for you by your friendly neighbourhood developer [Allen](https://github.com/allenbw)!  So without further ado lets :woman_juggling:! 
+
+## Featured
+So here are the raw numbers:
+* ManageIQ Core with [94 PRs merged][manageiq PRs merged] merged (awarded most overachieving!) :1st_place_medal:
```
Comment (syncrou) [inline site/_posts/2017-02-20-februarys-final-weeks-work.md]: @AllenBW - 94 didn't match the total when I clicked on the url.

File: site/_posts/2017-02-20-februarys-final-weeks-work.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## Fixed
+Sometimes an improvement is of the type two steps back.  In this case, a followup improvement of the type two steps forward is required, :bug: fixes! 
+The MIQ Service UI repo was brought a fix of this nature by [Chris](https://github.com/chalettu), [Fix for refresh of SUI $state.FeatureNav issue](https://github.com/ManageIQ/manageiq-ui-service/pull/496/files) which eliminated role based errors experienced when reloading the UI.
+In MIQ Classic repo broken automate icons [Fix broken Automate icons](https://github.com/ManageIQ/manageiq-ui-classic/pull/351) were fixed by [Eric](https://github.com/epwinchell).
+MIQ Core experienced the most hotly discussed bug fix (42 comments :eyes::feelsgood:) [Create a hawkular client for partial endpoints](https://github.com/ManageIQ/manageiq/pull/13814) authored by [Mooli](https://github.com/moolitayer)
+
+## New
+Hold the :phone:'s **newly introduced** REST API endpoints for service templates! This deeply pondered pr (Nov 2016  :scream_cat: ) [create service template REST api](https://github.com/ManageIQ/manageiq/pull/125940) was slain to completion by [Jillian](https://github.com/ManageIQ/manageiq/pull/12594).
```
Comment (syncrou) [inline site/_posts/2017-02-20-februarys-final-weeks-work.md]: @AllenBW - PR #125940 resulted in a `404`, also not sure if you meant to go directly to Jillian's github page or go to PR #12594 in the last url labeled `Jillian`

File: site/_posts/2017-02-20-februarys-final-weeks-work.md (hunk 1 above)
Comment (syncrou) [inline site/_posts/2017-02-20-februarys-final-weeks-work.md]: @AllenBW - 94 did not match the total I saw when clicking on the url

File: site/_posts/2017-02-20-februarys-final-weeks-work.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+MIQ Core experienced the most hotly discussed bug fix (42 comments :eyes::feelsgood:) [Create a hawkular client for partial endpoints](https://github.com/ManageIQ/manageiq/pull/13814) authored by [Mooli](https://github.com/moolitayer)
+
+## New
+Hold the :phone:'s **newly introduced** REST API endpoints for service templates! This deeply pondered pr (Nov 2016  :scream_cat: ) [create service template REST api](https://github.com/ManageIQ/manageiq/pull/125940) was slain to completion by [Jillian](https://github.com/ManageIQ/manageiq/pull/12594).  
+
+
+## Deleted
+Extending that :walking: motif from earlier, [Martin](https://github.com/martinpovolny) added to MIQ Classic a fantastic line reduction by employing mixins [Deduplicate show methods using mixins](https://github.com/ManageIQ/manageiq-ui-classic/pull/384).
+
+Another pr overflowing with juicy red minus numbers is [Delte ontap files](https://github.com/ManageIQ/manageiq-ui-classic/pull/392).  Brought to us by [Zita](https://github.com/ZitaNemeckova), this :hocho: removes all unused Ontap files :tada:.
```
Comment (syncrou) [inline site/_posts/2017-02-20-februarys-final-weeks-work.md]: s/Delte/Delete/

File: site/_posts/2017-02-20-februarys-final-weeks-work.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+## Wrapping up
+In conclusion, ya'll :rocket:. No really! Especially those who are absent mention in this LWIMIQ; which is only ever but a snapshot of all the awesome on goings of this repo. That being said, we always welcome your involvement, so head on over to [Current Issues](https://github.com/ManageIQ/manageiq/issues) to see where you can contribute.
+
+*Phew* that's it! It was an honour and pleasure to walk you around the contributes of the 170 developers working on ManageIQ Open-Source Management Platform, **STAY TUNED FOR MORE** or else  :taco::dancer:.  Oh and happy last week of February :feelsgood::wink: .
```
Comment (syncrou) [inline site/_posts/2017-02-20-februarys-final-weeks-work.md]: Possibly s/contributes/contributions/

**Comment by AllenBW** [comment]
@syncrou your swift well observed feedback is most appreciated! <3

File: site/_posts/2017-02-20-februarys-final-weeks-work.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+## Featured
+So here are the raw numbers:
+* ManageIQ Core with [94 PRs merged][manageiq PRs merged] merged (awarded most overachieving!) :1st_place_medal: 
+* ManageIQ Classic had [50 PRs][manageiq-ui-classic PRs merged] merged (eerily similar number as last week, for those of you keeing track, awarded most copycat!) :2nd_place_medal: 
+* ManageIQ Content saw [9 PRs][manageiq-content PRs merged] merged (awarded most generic! no really, checkout those prs!) :3rd_place_medal: 
+* ManageIQ Service UI saw [9 PRs][manageiq-ui-service PRs merged] merged (awarded most housekeeping!) :4th_place_medal:
+* Amazon provider had [4 PRs][manageiq-providers-amazon PRs merged] merged (awarded most resilient!) :5th_place_medal:
+* Azure provider had [3 PRs][manageiq-providers-azure PRs merged] merged (awarded most resilient!) :6 th_place_medal:
+
+But this :video_game: we play is not about the :trophy:'s, :taco:'s or :+1: its about the change, once such type of change being the Improvements!
```
Comment (imtayadeway) [inline site/_posts/2017-02-20-februarys-final-weeks-work.md]: s/once/one/ :heart:

File: site/_posts/2017-02-20-februarys-final-weeks-work.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+Hold the :phone:'s **newly introduced** REST API endpoints for service templates! This deeply pondered pr (Nov 2016  :scream_cat: ) [create service template REST api](https://github.com/ManageIQ/manageiq/pull/12594) was slain to completion by [Jillian](https://github.com/jntullo).  
+
+
+## Deleted
+Extending that :walking: motif from earlier, [Martin](https://github.com/martinpovolny) added to MIQ Classic a fantastic line reduction by employing mixins [Deduplicate show methods using mixins](https://github.com/ManageIQ/manageiq-ui-classic/pull/384).
+
+Another pr overflowing with juicy red minus numbers is [Delete ontap files](https://github.com/ManageIQ/manageiq-ui-classic/pull/392).  Brought to us by [Zita](https://github.com/ZitaNemeckova), this :hocho: removes all unused Ontap files :tada:.
+
+## Wrapping up
+In conclusion, ya'll :rocket:. No really! Especially those who are absent mention in this LWIMIQ; which is only ever but a snapshot of all the awesome on goings of this repo. That being said, we always welcome your involvement, so head on over to [Current Issues](https://github.com/ManageIQ/manageiq/issues) to see where you can contribute.
```
Comment (imtayadeway) [inline site/_posts/2017-02-20-februarys-final-weeks-work.md]: :heart:

**Comment by imtayadeway** [comment]
What a lovely issue! Lil' change suggested above, otherwise :shipit:

File: site/_posts/2017-02-20-februarys-final-weeks-work.md (hunk 5 above)
Comment (AllenBW) [inline site/_posts/2017-02-20-februarys-final-weeks-work.md]: OOOOO :eyes:

**Comment by AllenBW** [comment]
:bowing_woman: <3 @imtayadeway thanks! updated!

**Comment by JPrause** [comment]
@cybette or @hayesr when you are available, can one of you merge this finely crafted blog entry.

File: site/_posts/2017-02-20-februarys-final-weeks-work.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
+---
+title: "February's Final Week's Work"
```
Comment (cybette) [inline site/_posts/2017-02-20-februarys-final-weeks-work.md]: The LWIMIQ posts have "Last Week in ManageIQ" in the title, i.e.

`title: "Last Week in ManageIQ - February's Final Week's Work"`

File: site/_posts/2017-02-20-februarys-final-weeks-work.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+Woah first last week of February for 2017, this is :taco::dancer: **huge**. *Almost* as :taco::dancer: huge as the LQIMIQ low down prepared for you by your friendly neighbourhood developer [Allen](https://github.com/allenbw)!  So without further ado lets :woman_juggling:! 
+
+## Featured
+So here are the raw numbers:
+* ManageIQ Core with [94 PRs merged][manageiq PRs merged] merged (awarded most overachieving!) :1st_place_medal: 
+* ManageIQ Classic had [50 PRs][manageiq-ui-classic PRs merged] merged (eerily similar number as last week, for those of you keeing track, awarded most copycat!) :2nd_place_medal: 
+* ManageIQ Content saw [9 PRs][manageiq-content PRs merged] merged (awarded most generic! no really, checkout those prs!) :3rd_place_medal: 
+* ManageIQ Service UI saw [9 PRs][manageiq-ui-service PRs merged] merged (awarded most housekeeping!) :4th_place_medal:
+* Amazon provider had [4 PRs][manageiq-providers-amazon PRs merged] merged (awarded most resilient!) :5th_place_medal:
+* Azure provider had [3 PRs][manageiq-providers-azure PRs merged] merged (awarded most resilient!) :6 th_place_medal:
```
Comment (cybette) [inline site/_posts/2017-02-20-februarys-final-weeks-work.md]: Not sure if :4th_place_medal: :5th_place_medal: and :6th_place_medal: are valid emojis, at least they are not showing up for me...

Maybe :trophy: or :gem: ?

**Comment by cybette** [comment]
Sorry for last minute comments, looks great @AllenBW ! Will merge after final changes.

File: site/_posts/2017-02-20-februarys-final-weeks-work.md (hunk 8 above)
Comment (AllenBW) [inline site/_posts/2017-02-20-februarys-final-weeks-work.md]: they aren't, its intentional kinda a womp womp

Comment (cybette) [inline site/_posts/2017-02-20-februarys-final-weeks-work.md]: @AllenBW I thought that might be the case :) How about the space in :6 th_place_medal: , is that intentional as well?

Comment (AllenBW) [inline site/_posts/2017-02-20-februarys-final-weeks-work.md]: @cybette updated for clarity, womp womp emojis fo LIFE :metal:

**Pull request merged**